### PR TITLE
add migaloo-terra channels

### PR DESCRIPTION
**Pull request by fragwuerdig**

### CODE REVIEW

File: _IBC/terra-migaloo.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "terra",
```
Comment (JeremyParish69) [inline _IBC/terra-migaloo.json:4]: please reverse the order of chain_1 and chain_2 (because terra comes later, alphabetically). And the file name, too

Comment (fragwuerdig) [reply _IBC/terra-migaloo.json]: Done

**Comment by fragwuerdig** [comment]
Please, advice on how to add a second/different client/connection pair for the same pair of chains.

**Comment by JeremyParish69** [comment]
> Please, advice on how to add a second/different client/connection pair for the same set of chains.

You would add additional objects inside the 'channels' array in the appropriate file.

**Pull request merged**